spot-electron/src/bt-beacon/btbeacon.js

```diff
@@ -17,8 +19,9 @@ class BTBeacon {
      */
     constructor() {
         this.region = config.getValue('beacon.region');
+        const isWin = platform === 'win32';
```

Comment:
Instead check explicitly if the platform is darwin, since this is also not supported on Linux.

Reply:
Is the check necessary though? Isn't `daemon` undefined?

Reply:
Default was {}, changed to export undefined and removed this logic from here